cli/tests/test_global_opts.rs

```diff
@@ -96,7 +96,10 @@ fn test_no_subcommand() {
     │  (empty) (no description set)
     ~
     "###);
-    insta::assert_snapshot!(test_env.jj_cmd_success(&repo_path, &["-r", "show"]), @r###"
```

Comment:
This is the test that `-r show` isn't parsed as `-r -- show`. (I don't remember why I added both `log` and `show`, though.)

Maybe add new test for array default commands?

Reply:
Done.